test/format.spec.js

```diff
@@ -0,0 +1,104 @@
+import { expect } from "chai";
+import { format } from "../src/format";
+import { notInGivenKeys, transformToStrs } from "./helper";
+import romanizedExpectedKeys from "./layouts/romanized/helpers/expected_keys";
+import romanizedTestKeys from "./layouts/romanized/helpers/test_keys";
+import traditionalExpectedKeys from "./layouts/traditional/helpers/expected_keys";
+import traditionalTestKeys from "./layouts/traditional/helpers/test_keys";
+
+describe("format", () => {
+  describe("format(..., option)", () => {
+    it("should not throw", () => {
+      const layouts = ["romanized", "traditional"];
+
+      expect(() => format("trtr")).to.not.throw();
+      expect(() =>
+        layouts.forEach((layout) => format("trtr", { layout: layout }))
+      ).to.not.throw();
+    });
+
+    it("should throw", () => {
+      expect(() => format("trtr", { layout: "not a layout" })).to.throw();
+    });
+  });
+
+  describe("format(..., romanized)", () => {
+    const romanizedOption = { layout: "romanized" };
+
+    it("should return mapped romanized string", () => {
+      const inputStrs = transformToStrs(romanizedTestKeys);
+      const expectedStrs = transformToStrs(romanizedExpectedKeys);
+
+      const returned = inputStrs.map((value) => format(value, romanizedOption));
+
+      expect(returned).to.deep.equal(expectedStrs);
+    });
+
+    it("should return input string if could not map to romanized.", () => {
+      const inputKeys = notInGivenKeys(romanizedTestKeys);
+      const inputStrs = transformToStrs(inputKeys);
```

Comment:
two to three asserts here should be enough. To make it interesting you can try to use non english unicode text as well.